otter/test/models/test_intents.py

```diff
@@ -15,6 +19,32 @@ def test_perform(self):
                                modifier=lambda g, o: 'new state')
         dispatcher = TypeDispatcher({
             ModifyGroupState: perform_modify_group_state})
-        d = perform(dispatcher, Effect(mgs))
-        self.assertEqual(self.successResultOf(d), 'new state')
+        result = sync_perform(dispatcher, Effect(mgs))
+        self.assertEqual(result, 'new state')
         self.assertEqual(group.modify_state_values, ['new state'])
+
+
+class GetScalingGroupInfoTests(SynchronousTestCase):
+    """Tests for :obj:`GetScalingGroupInfo`."""
+    def test_perform(self):
+        """Performing returns the group, the state, and the launch config."""
+        log = mock_log()
+        state = object()
+        lc = object()
+        group = mock_group(state)
+        group.view_state.return_value = succeed(state)
+        group.view_launch_config.return_value = succeed(lc)
+
+        data = {('00', 'g1'): group}
```

Comment:
Nice!  To make sure it has the right tenant ID and group ID?  Non-blocking just-curiosity question - why the dictionary here, but the assertion about the log below?  Are there going to be more groups here in a future PR?